connector_ebisumart/models/ebisumart_sale_order.py

```diff
@@ -117,6 +117,7 @@ def after_import(self, ebisumart_record, backend_record):
             'purchase_id': po.id,
         }
         inv = self.env['account.invoice'].create(po_invoice)
+        inv.write({'date_invoice': self.ebisumart_send_date})
```

Comment:
In this method, the previous day is recorded due to date and datetime fields. In the case of the UTC+9 time zone, even though ebisumart_send_date is a datetime field with October 18th 00:00:00, in the database it is recorded as October 17th. 
When this value is passed to a Date field, it gets recorded as October 17th. 
This issue does not occur with Customer Invoice